Inc/cli/screen.hpp

```diff
@@ -18,14 +18,16 @@ class Screen {
     QString screenName;
     std::vector<std::shared_ptr<Screen>> childScreens;
     std::shared_ptr<Screen> parentScreen;
-    std::vector<Option> screenOptions;
+    std::vector<std::shared_ptr<Option>> screenOptions;
     int activeOption = -1;
     std::vector<QString> notifications = {};
 
     Screen(QString name, std::shared_ptr<Screen> parentScreen);
 
     void addScreenOption(const QString &name, std::function<void()> const& action);
 
+    void addScreenOption(const QString &name, std::function<void()> const& action, bool isSwitcher, bool switcher);
```

Comment:
Consider removing the unnecessary parameter `isSwitcher` in the declaration of the overloaded method `addScreenOption`, because this parameter also uniquely defines that the option is a switcher.

```suggestion
    void addScreenOption(const QString &name, std::function<void()> const& action, bool switcher);
```

Reply:
Fixed in 67a0ea33ae94f9484568675c3ce69e64bd2abc9d